subprojects/build-cache-http/src/main/java/org/gradle/caching/http/internal/HttpBuildCacheService.java

```diff
@@ -180,7 +180,7 @@ public boolean isStreaming() {
             if (!isHttpSuccess(statusCode)) {
                 String defaultMessage = String.format("Storing entry at '%s' response status %d: %s", safeUri(uri), statusCode, statusLine.getReasonPhrase());
                 if (isRedirect(statusCode)) {
-                    handleRedirect(uri, response, statusCode, defaultMessage);
+                    handleRedirect(uri, response, statusCode, defaultMessage, "storing entry to");
```

Comment:
I think this is "storing entry at" not "to".

Reply:
fixed.